syntax = "proto3";

package relighting;

service RelightingService {
  rpc Relight (RelightRequest) returns (RelightResponse);
}

message RelightRequest {
  bytes image_data = 1;
  bytes mask_data = 2;
  bytes json_data = 3;
}

message RelightResponse {
  bytes processed_image_data = 1;
}
